src/runtime/basic-runtime.ts

```diff
@@ -207,27 +220,28 @@ export class BasicRuntime implements BasicSubscriptions {
     publisherProvidedId?: string;
   }): void {
     this.enableDefaultMeteringHandler_ = !disableDefaultMeteringHandler;
-    this.pageConfigWriter_ = new PageConfigWriter(this.doc_);
     this.publisherProvidedId_ = publisherProvidedId;
-    this.pageConfigWriter_
-      .writeConfigWhenReady({
-        type,
-        isAccessibleForFree,
-        isPartOfType,
-        isPartOfProductId,
-      })
-      .then(() => {
-        this.pageConfigWriter_ = null;
-        this.configured_(true);
-      });
+    const isOpenAccess = isOpenAccessProductId(isPartOfProductId);
+    let isClosable = isAccessibleForFree;
+    // Only overwrite isClosable if product is openaccess, else leave undefined.
+    if (isOpenAccess) {
+      isClosable ??= true;
+    }
+    isAccessibleForFree ??= isOpenAccess;
```

Comment:
Should this be left undefined if product is not openaccess too?

Reply:
resolving as discussed in b/280335205.